### DISCUSSION TITLE
Economy plugins and Towny

### DISCUSSION THREAD

**Discussion by InsaniteaTV** [TownyAdvanced/Towny#5015] [Q&A] [answered]
I see a lot of the stuff on the wiki about economy, but not if there's a need for a separate economy plugin from the Towny plugin itself. If this is answered anywhere on the wiki sorry for asking, but do you need a separate one?

**Comment by LlmDl** [comment, marked as answer]
You will need an economy plugin if you want the economy-related features. Towny doesn't have an economy plugin built-in. https://github.com/TownyAdvanced/Towny/wiki/Frequently-Asked-Questions has a few recommendations for eco plugins.
